### DISCUSSION TITLE
Custom animate className won't define

### DISCUSSION THREAD

**Discussion by f3bw** [tailwindlabs/tailwindcss#11611] [Help]
Good day, I am trying to put a custom animation on an element when it is opened. But I get the error `Classname 'data-[state=open]:animate-fadein' is not a Tailwind CSS class!`

Here is my tailwind.config.js file content:
```
module.exports = {
  mode: 'jit',
  content: ['./app/**/*.{js,ts,jsx,tsx}', './src/**/*.{js,ts,jsx,tsx}, '],
  darkMode: 'media', 
  theme: {
    extend: {},
  },
  variants: {
    extend: {
      keyframes: {
        fadeIn: {
          from: { opacity: '0' },
          to: { opacity: '1' },
        },
        fadeOut: {
          from: { opacity: '1' },
          to: { opacity: '0' },
        },
      },
      animation: {
        fadein: 'fadeIn 150ms ease-in',
        fadeout: 'fadeOut 150ms ease-in',
      },
    },
  },
  plugins: [],
}

```

And I try to apply it like this:
```
<Popover.Content
        className='data-[state=open]:animate-fadein w-[260px] rounded-xl  bg-[rgba(220,220,220,.4)] p-5 py-2.5 font-medium text-black/40  will-change-[transform,opacity]'
        sideOffset={5}
>
```

The location of the file is ./src/components/About/About.jsx, so it should fall under './src/**/*.{js,ts,jsx,tsx},

Could anyone help me out?

Cheers!

**Comment by wongjn** [comment]
Where is the error coming from?

**Reply by f3bw** [reply]
The error is coming from the file where the Popover.Content part is located. So not from the tailwind.config file

**Reply by wongjn** [reply]
What is emitting the error? Consider providing a remotely-hosted git repository that reproduces the unexpected behavior.
